src/components/org/OrgChart/components/Aside.tsx

```diff
@@ -69,7 +69,7 @@ function Aside<T>() {
                     key={card.id}
                     node={card}
                     x={startX + i * (cardWidth + cardMargin)}
-                    y={(rowNo + 1) * (rowMargin - 20) + 20}
+                    y={(rowNo + 1) * rowMargin + 24}
```

Comment:
Think it's best to use  y={(rowNo + 1) * (rowMargin - 20) + 20} for now, so the spacing between the aside rows don't get too big. Maybe change "+ 20" to "+24"?

Reply:
its now "(rowMargin - 20) + 24"